job_bundles/afterfx_render_one_task/template.yaml

```diff
@@ -73,15 +82,16 @@ steps:
   script:
     actions:
       onRun:
-        command: aerender
-        args:
-        - "-project"
-        - "{{Param.ProjectFile}}"
-        - "-comp"
-        - "{{Param.CompName}}"
-        - "-output"
-        - "{{Param.OutputDirectory}}"
-        - "-s"
-        - "{{Param.StartFrame}}"
-        - "-e"
-        - "{{Param.EndFrame}}"
+        command: bash
+        args: ['{{Task.File.Run}}']
+    embeddedFiles:
+    - name: Run
+      type: TEXT
+      data: |
+        mkdir -p '{{Param.OutputDirectory}}'
```

Comment:
Since we're now running bash scripts, you'll want to add `set -euo pipefail` to the top of this bash script to ensure that an `aerender` failure bubbles up to the bash script failure.